tiny/src/cmd.rs

```diff
@@ -422,7 +422,9 @@ fn names(args: CmdArgs) {
     };
 
     if let MsgSource::Chan { ref serv, ref chan } = src {
-        let nicks_vec = client.get_chan_nicks(chan);
+        use lexical_sort::{lexical_only_alnum_cmp, StringSort};
```

Comment:
Scoped 'use...' to limit the scope of the StringSort trait? Not sure if this is an anti-pattern or accepted as "efficient" or something.

Reply:
I think this is fine.

Did you check how much the release binary size changes with this library?